src/components/screens/Authenticator/modules/OtpCode/AuthController.js

```diff
@@ -0,0 +1,34 @@
+import { VuexModule, Action, Module } from 'vuex-class-modules';
```

Comment:
This and `EmailCode/AuthController.js` have the same code. Can we use shared controllers? Why we should duplicate logic? I see, it is very simple, but I don't understand, why we can't share and reuse that.

Reply:
We can, as vault